components/screens/interfaces.ts

```diff
@@ -0,0 +1,15 @@
+export type ScreenNames = 
+  | 'Profile'
+  | 'Centers'
+  | 'About'
+  | 'Gallery'
+  | 'Career'
+  | 'Contact'
+  | 'Terms'
+  | 'Refund'
+  | 'Privacy'
+  | 'Logout';
+
+export interface NavigationProp {
+  navigate: (screen: ScreenNames) => void;
+}
```

Comment:
create a file types.ts dir and move it to the file so that we could have separate directory for the dedicated types

Reply:
move this file to types dir